### PR TITLE
Issue-377 remove reference to transactable

### PR DESCRIPTION
**Pull request by lemingos**
Fix for #377

### CODE REVIEW

File: marketplace_builder/views/pages/tutorials/amp/feeding-data-amp-pages-graphql.liquid [hunk 1]
```diff
@@ -82,18 +82,18 @@ format: json
 ```
 ```liquid
 {% raw %}
-{% query_graph 'get_listings', result_name: g %}
-{% assign results = g.listings.results %}
+{% query_graph 'get_products', result_name: g %}
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/amp/feeding-data-amp-pages-graphql.liquid]:
```suggestion
{% query_graph 'get_products', result_name: 'g' %}
```

**Comment by pavelloz** [comment]
Does this article have corresponding example?
If yes, would be nice to update it as well.

If not, worth considering copying code to prove its working, while its still fresh :)

**Comment by lemingos** [comment]
> Does this article have corresponding example?

I don't think it has one.

> If yes, would be nice to update it as well.

Yes maybe the author of AMP pages article could create one? :)

**Review by diana-lakatos — APPROVED** [review]
Reviewed and edited, thank you!

**Comment by pavelloz** [comment]
Sadly author of amp pages doesnt know what author of this article wanted to express and doesnt have files to copy (im assuming everything written in documentation is checked before put online)

**Comment by pavelloz** [comment]
I would say complete documentation looks like this:

1) live example (in a module) - it helps a lot for people that want to see the generated html, etc.
2) example source code in our examples repo - since you need to have the thing working before describing it, its a no brainer
3) article - describing why and in what order thing is developed
4) e2e tests for example - those are run on every platform merge, before anything gets to production, so having feature tested makes a great proof that the thing is actually still working

I would say this is a good inspiration: https://github.com/mdyd-dev/nearme-documentation/issues/393#issuecomment-463651846


I understand that writing e2e in js is not the favorite and its bonus work, so i would say we (me & Jacek) can handle that for now, but there is no excuse to not do the first 2 steps if 3rd is done - most of the work had to be done anyways.

**Comment by diana-lakatos** [comment]
Closing this issue now to remove deprecated feature from documentation. Opened ticket about adding example: #421

**Pull request merged**